File: submit/src/main/java/material/com/submit/prestenter/SubmitPresenter.java
package material.com.submit.prestenter;

import material.com.submit.ui.SubmitContact;

/**
 * Created by cangwang on 2017/3/30.
 */

public class SubmitPresenter implements SubmitContact.Presenter{

    private SubmitContact.View mSubmitView;

    public SubmitPresenter(SubmitContact.View view){
        mSubmitView = view;
        mSubmitView.setPresenter(this);
    }

    @Override
    public void onResume() {

    }

    @Override
    public void onPause() {

    }

    @Override
    public void onDestroy() {

    }

    @Override
    public void start() {

    }
}

File: submit/src/main/java/material/com/submit/prestenter/SubmitPresenter.kt
package material.com.submit.prestenter

import material.com.submit.ui.SubmitContact

/**
 * Created by cangwang on 2017/3/30.
 */

class SubmitPresenter(private val mSubmitView: SubmitContact.View) : SubmitContact.Presenter {

    init {
        mSubmitView.setPresenter(this)
    }

    override fun onResume() {

    }

    override fun onPause() {

    }

    override fun onDestroy() {

    }

    override fun start() {

    }
}
